README.md

```diff
@@ -11,6 +11,8 @@ To begin, clone this repository. `cd` into your chosen folder, and run the follo
 
 Selenium server and Chromedriver will also be downloaded into `/selenium`. 
 
+Currently Using https://www.npmjs.com/package/selenium-download version: 2.0.10
```

Comment:
Would be useful to list the Chromedriver and Selenium versions that are installed, as that's what we're more interested in:

```
[testium] grabbing selenium chromedriver 2.29
[testium] grabbing selenium standalone server 3.4.0
```